doc/src/features/ExcelImport.md

```diff
@@ -8,24 +8,25 @@ In the middle of the site is a form with a blue upload button.
 
 It is possible to upload an Excel file there.
 
-Non-Excel files or old format Excel files (older than 2007) will produce errors after uploading.
+Non-Excel files or old-format Excel files (older than 2007) will produce errors after uploading.
 
 The Excel file has to be in <a href="../assets/Import_Standart.xlsx" download>this correct format</a>.
 
-The First Name (Vorname) has to be between 2 and 255 characters.
+The first name (Vorname) has to be between 2 and 255 characters.
 
-The Last Name (Nachname) has to be between 2 and 255 characters.
+The last name (Nachname) has to be between 2 and 255 characters.
 
-The training direction / course of study (Studiengang) has to be one of these: BSC, BWI, VI, FISI.
+The training direction or course of study (Studiengang) has to be one of these: BSC, BWI, VI, or FISI.
 
-The year (Jahrgang) has to be in this format: XX/XX Example: 20/24 for someone that started training in 2020 and finishes in 2024.
+The year (Jahrgang) has to be in this format: XX/XX Example: 20/24 for someone who started training in 2020 and finishes in 2024.
 
-The lecture days (Vorlesungstage) have to be in the standard german short notation of Days ([see Table below](#english-days-to-german-short-notation)). 
-The Days can be chained together by plus signs, for Example: Mo + Di + Mi
+The lecture days (Vorlesungstage) have to be in the standard German short notation of Days ([see Table below](#english-days-to-german-short-notation)).
+The days can be chained together by plus signs, for example: Mo + Di + Mi.
 
-If the Upload is successful, a success message will appear. If it's not, a red error will appear.
+If the upload is successful, a success message will appear. If it's not, a red error will appear.
 
-### English days to german short notation
+### English days to German short notation
 | English | Monday | Tuesday | Wednesday | Thursday | Friday | Saturday |
 |---------|--------|---------|-----------|----------|--------|----------|
-| German  | Mo     | Di      | Mi        | Do       | Fr     | Sa       |
\ No newline at end of file
+| German | Mo | Di | Mi | Do | Fr | Sa |
```

Comment:
My IDE says "Table is not correctly formatted " with this formatting.
As this reflects  table, we should keep the old formatting